docs/topics/3rd_party_voice_clients(experiment).md

```diff
@@ -292,10 +292,10 @@ if (region is None):
 
 ### Getting entity from cache
 
-At cases you might need rich information about entities, but you only have their id available. Even tho most objects
-expose entity access too, like, they have `channel_id` attribute and a `channel` property, this may not be enough.
+In certain cases you might need rich information about entities, but you only have their ID available.
+Even tho most objects will expose entity access too, e.g. they have `channel_id` attribute and a `channel` property, this may not be enough.
 
-At these cases, use `USERS`, `GUILDS`, `CHANNELS` caches to access the entities.
+In these cases use `USERS`, `GUILDS`, `CHANNELS` caches to access the entities.
 
 ```py
 channel = CHANNESL.get(channel_id, None)
```

Comment:
I assume this is a typo? Waiting for verification.

Reply:
Indeed it is.